lib/tasks/dev.rake

```diff
@@ -0,0 +1,89 @@
+desc "Fill the database tables with some sample data"
+task({ :sample_data => :environment}) do
+  
+  User.destroy_all
+  Loan.destroy_all
+  Adjustment.destroy_all
+
+  # Create Sample Users
+    #  email                  :string           default(""), not null
+    #  encrypted_password     :string           default(""), not null
+  email = ["alice@example.com", "bob@example.com", "carol@example.com"]
+
+  3.times do |count|
+    user = User.new
+    user.email = email.at(count)
+    user.password = "password"
+    user.save
+  end
+
+  p "Added #{User.count} Users"
+
+  # Create Sample Loans
+    #  current_balance     :float
+    #  original_amount     :float
+    #  interest_rate       :float
+    #  periods_in_year     :integer
+    #  user_id             :integer
+    #  loan_name           :string
+    #  monthly_min_payment :float
+
+    #cur_bal = [1000, 5000, 10000, 15000, 20000, 25000, 30000, 35000, 40000]
+    org_bal = [1000, 5000, 10000, 15000, 20000, 25000, 30000, 35000, 40000]
+    int_rate = [2, 3, 4, 5, 6]
+    payback_per = [2, 4, 6]
+    per_in_year = 12
+    users = User.all
+    l_name = ["Car", "Car 2", "Student Loan", "Grad School", "Credit Card"]
+
+    10.times do
+      loan = Loan.new
+      loan.current_balance = org_bal.sample
+      loan.original_amount = loan.current_balance
+      loan.interest_rate = int_rate.sample
+      loan.periods_in_year = per_in_year
+      loan.user_id = users.sample.id
```

Comment:
How about

```suggestion
      loan.user = users.sample
```

? We are allowed to do that since we're using `belongs_to`, and since we've already created the `ActiveRecord` instance there's no memory savings of dealing with raw foreign keys. And this reads a bit more nicely / allows us to think at a higher/more appropriate domain level.

Reply:
That makes sense, and agree, it is more readable and better portrays what I am actually trying to assign here.